doc/contributing/submitting-changes.chapter.md

```diff
@@ -21,6 +21,10 @@
 
 - Make commits of logical units.
 
+- Using a separate commit, register into the `.git-blame-ignore-revs` file any commits that you believe may be distracting when using `git blame`. The most common examples include commits that introduce purely cosmetic changes, such as commits that only rename symbols or adjust indentation.
```

Comment:
I think we should add to only do this for bigger changes to not collect very small commits.